docs/source/getting-started/first-feature.rst

```diff
@@ -175,24 +179,22 @@ You can see we simply follow the formatting of the previous panels and include t
 That is all for the implementation!
 Let's give the new functionality a spin.
 
-Running the New App
-===================
+Running the New Feature
+=======================
 
-To start up the new app, open up your terminal application.
-First make sure your previously run commands are terminated, in particular the ``daml-start.sh`` command.
-You can do this by hitting ``Ctrl-C`` in the terminal window where you ran the command.
-This shuts down the previous instances of the sandbox and JSON API server: it is important that we start our new app with new instances of these components.
+We need to terminate the previous ``./daml-start.ch`` process and run it again, as we need to have a Sandbox instance with a DAR file containing the new feature. As a reminder, by running the ``./daml-start.ch`` again we will 
 
-Having done that, first restart the DAML sandbox and JSON API server in the root ``create-daml-app`` folder::
+  - Compile our DAML code into a *DAR file contatining the new feature*
+  - Run a fresh instance of the *Sandbox with the new DAR file*
+  - Start the HTTP JSON API 
 
-    ./daml-start.sh
+First, navigate to the terminal window where the ``daml-start.sh`` is running and terminate the active process by hitting ``Ctrl-C``. This shuts down the previous instances of the sandbox. Next in the root ``create-daml-app`` folder run ``./daml-start.sh``.
```

Comment:
@cocreature This is the new explanation that also serves as a refresher on what ./daml.start.sh does (from the "Getting started with DAML" section)

Reply:
:+1: